src/NuGet.Clients/NuGet.PackageManagement.UI/Resources/Resources.xaml.cs

```diff
@@ -19,7 +19,7 @@ private void PackageIconImage_ImageFailed(object sender, ExceptionRoutedEventArg
             var image = sender as Image;
 
             e.Handled = true; // don't repropagate the event
-            BindingExpression binding = BindingOperations.GetBindingExpression(image, Image.SourceProperty);
+            MultiBindingExpression binding = BindingOperations.GetMultiBindingExpression(image, Image.SourceProperty);
             if (binding != null && binding.Status != BindingStatus.Detached)
```

Comment:
I think you can safely remove `binding.Status != BindingStatus.Detached` condition because [`MultiBindingExpression.UpdateTarget()`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.multibindingexpression.updatetarget?view=netframework-4.8) doesn't throw exception if the binding status is` Detached` like [`BindingExpression.UpdateTarget()`](https://docs.microsoft.com/en-us/dotnet/api/system.windows.data.bindingexpression.updatetarget?view=netframework-4.8). May be `binding?.UpdateTarget()` is sufficient.

Reply:
Good catch! But, I also need to check the detached status for each BindingExpression inside the MultiBindingExpression